src/shareon.js

```diff
@@ -18,7 +18,7 @@ import "./shareon.css";
  */
 const urlBuilderMap = {
   facebook: (d) => `https://www.facebook.com/sharer/sharer.php?u=${d.url}${d.hashtags? `&hashtag=%23${d.hashtags.split('%2C')[0]}` : ''}`,
-  linkedin: (d) => `https://www.linkedin.com/sharing/share-offsite/?url=${d.url}`,
+  linkedin: (d) => `https://www.linkedin.com/sharing/share-offsite/?url=${d.url}${encodeURIComponent('?latest')}`,
```

Comment:
Is there a reason why you have replaced `&latest` with this? Does the query parameter need to appear on the URL being shared (e.g., `//shareon.js.org/?latest`) or on the final LinkedIn URL?

Reply:
Just did more testing and the solution is to append the current time to the end of the url (encoded) so that LinkedIn regards it as a new url altogether and fetches the latest preview.

Reply:
Just added ${encodeURIComponent('?' + new Date().getTime())}